docs/devGuide/bootcamp/exploreMarkBind.md

```diff
@@ -0,0 +1,242 @@
+
+Now, let's try editing the `index.md` file to add some content of our own!
+
+1. Open the `index.md` file in your `mb-dev-xxx` folder.
+1. Replace the wording `Landing Page Title` with `MarkBind Developer Test Site`.
+1. Modify or add some content utilizing one or more of the following syntaxes:
+    - [Text Styles]({{baseUrl}}/userGuide/formattingContents.html#text-styles)
+    - [Code Block]({{baseUrl}}/userGuide/formattingContents.html#code)
+    - [Emoji]({{baseUrl}}/userGuide/formattingContents.html#emoji)
+    - etc.
+1. Modify or add some content utilizing one or more of the following components:
+    - [Box]({{baseUrl}}/userGuide/components/presentation.html#boxes)
+    - [Panel]({{baseUrl}}/userGuide/components/presentation.html#panels)
+    - [Tooltip]({{baseUrl}}/userGuide/components/popups.html#tooltips)
+    - etc.
+
+<box type="important" light>
+
+**Checkpoint**
+
+With live preview, you should be able to see the changes you have made to the site after saving. Check that the following are true:
+- [ ] Serve the site by running `markbind serve -d`
+- [ ] Make changes to the `index.md` file and save
+- [ ] See the changes reflected in the live preview, with the correct rendering of the syntax and components
+
+You now have created content with MarkBind!
+
+</box>
+
+## Modify Site Structure and Configuration
+
+There are a few ways to modify the site's (as well as the page's) structure and configuration. We will cover the following:
+- [Frontmatter]({{baseUrl}}/userGuide/tweakingThePageStructure.html#frontmatter)
+- [Layouts]({{baseUrl}}/userGuide/tweakingThePageStructure.html#layouts)
+- [site.json]({{baseUrl}}/userGuide/siteJsonFile.html)
+
+Let's continue with the test site we created earlier and modify the `index.md` file via frontmatter.
+
+1. Open the `index.md` file in your `mb-dev-xxx` folder.
+1. Change the `pageNavTitle` in the frontmatter from `Chapters of This Page` to `Index Page`.
+1. Change the `pageNav` in the frontmatter from `2` to `1`.
+
+<box type="important" light>
+
+**Checkpoint**
+
+Ensure that you are still serving the site with `markbind serve -d`. Check that the following are true:
+- [ ] The rendered page navigation title is changed to `Index Page`
+- [ ] The rendered page navigation content only shows `H1` headings
+
+</box>
+
+Another commonly adjusted structure is the layout of the site (especially, the site's top header and the left-hand-side navigation).
+
+1. Open the `_markbind/layouts/default.md` file in your `mb-dev-xxx` folder.
+1. Change `Your Logo` to `MarkBind Developer Test Site`.
+1. Change `Home :house:` to `My Test Playground :computer:`.
+
+<box type="important" light>
+
+**Checkpoint**
+
+Ensure that you are still serving the site with `markbind serve -d`. Check that the following are true:
+- [ ] The rendered page logo in the header is changed to `MarkBind Developer Test Site`
+- [ ] The rendered site navigation title is changed to `My Test Playground` :computer:
```

Comment:
How about adding some screenshots in the checkpoint area to help user better visualize what changes to look out for? 

Same for the above 2 checkpoints as well :)